config/dev.exs

```diff
@@ -61,3 +61,12 @@ config :code_corps, CodeCorps.Mailer,
 
 config :code_corps,
   postmark_receipt_template: "123"
+
+# If the dev environment has no CLOUDEX_API_KEY set, we want the app to still run,
+# with cloudex in test API mode
+if System.get_env("CLOUDEX_API_KEY") == nil do
+  config :cloudex, :cloudinary_api, Cloudex.CloudinaryApi.Test
+  config :cloudex, api_key: "test_key", secret: "test_secret", cloud_name: "test_cloud_name"
+
+  IO.puts("NOTE: No Cloudex configuration found. Cloudex is runnning in test mode.")
```

Comment:
It's just a random value. We need to provide a value, any value, or Cloudex raises an error on startup.